task/runner.go

```diff
@@ -66,3 +69,14 @@ func (r *Runner) getShouldRun() bool {
 	minutesElapsed := now.Sub(lastCheckedAt).Minutes()
 	return minutesElapsed > minimumMinutesBetweenChecks
 }
+
+func (r *Runner) shouldRunUpdateChecks() bool {
```

Comment:
add a comment - return whether to run the update checks. These are disabled when running queries in batch mode.

add a similar comment for shouldRun